### PR TITLE
Updata_service.md

### PR DESCRIPTION
**Pull request by yunxji**
Added some TKE Service annotations.
/assign @tengqm

### CODE REVIEW

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -870,6 +879,40 @@ in those modified security groups.
 
 {{< /note >}}
 
+#### Other CLB annotations on TKE
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/service.md]: The first time you use an acronym like TKE, please spell it out.

File: content/en/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -870,6 +879,40 @@ in those modified security groups.
...
+
+There are other annotations for managing Cloud Load Balancer on TKE as described below.
+
+```yaml
+    metadata:
+      name: my-service
+      annotations:
+        service.kubernetes.io/qcloud-loadbalancer-backends-label： `key in (value1, value2)`
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/service.md]: I'm not sure how backtiks  work in side a code block, please avoid using them if you are unsure about it.

File: content/en/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -870,6 +879,40 @@ in those modified security groups.
...
+        # Bind Loadbalancers with speicfied nodes
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/service.md]: the convention is to have comments BEFORE the actual stanza.

Comment (yunxji) [reply content/en/docs/concepts/services-networking/service.md]: Does this suggest that the annotation description should be placed before the annotation?

The format here is a reference to AWS.
https://kubernetes.io/docs/concepts/services-networking/service/#other-elb-annotations

Comment (yunxji) [reply content/en/docs/concepts/services-networking/service.md]: > the convention is to have comments BEFORE the actual stanza.

Does this suggest that the annotation description should be placed before the annotation?

The format here is a reference to AWS.
https://kubernetes.io/docs/concepts/services-networking/service/#other-elb-annotations

File: content/en/docs/concepts/services-networking/service.md (hunk 1 above)
Comment (yunxji) [reply content/en/docs/concepts/services-networking/service.md]: > The first time you use an acronym like TKE, please spell it out.

Thanks for your suggestions, I have modified the content according to your suggestions.

File: content/en/docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -870,6 +879,42 @@ in those modified security groups.
...
+        service.kubernetes.io/tke-existed-lbid：lb-6swtxxxx
+        
+        # Custom parameters for the load balancer (LB), does not support modification of LB type yet
+        service.kubernetes.io/service.extensiveParameters: ""
+        
+        # Custom parameters for the LB listener 
+        service.kubernetes.io/service.listenerParameters: ""
+        
+        # Specifies the type of CLB: Cloud Load Balancer（default）or Classic Cloud Load Balancer
+        # Valid values: classic（Classic Cloud Load Balancer）,any other fields or not specified(Cloud Load Balancer)
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/service.md]: Line 902-903 does not make senses, pls revise

**Pull request merged**